lib/redmine/menu_manager/menu_helper.rb

```diff
@@ -265,4 +265,13 @@ def allowed_node?(node, user, project)
       return true
     end
   end
+
+  def visible_node?(menu, node)
+    if OpenProject::Configuration['hidden_menu_items'].length > 0
+      hidden_nodes = OpenProject::Configuration['hidden_menu_items'][menu.to_s] || []
```

Comment:
Also, you should use the getter here. I.e. `OpenProject::Configuration.hidden_menu_items`.